### DISCUSSION TITLE
Why do transport nodes need to be stationary?  Why not Messenger nodes?

### DISCUSSION THREAD

**Discussion by ReconPirate** [markqvist/Reticulum#489] [Ideas]
Okay basic long and short, I know that the reason they are meant to be stationary is so that they can be used as steady nodes for communication between other nodes and used to store and pass messages between other and eventually make up links to be more reliable and cover a larger distance.  

 Besides the more reliable links and greater infrastructure, my main question is what would break if you were to have a few transition nodes in your area be mobile, in theory all other nodes they passed would be able to pass messages 

>.> but  really this makes me think that this wasn't originally in mind
 "Every Transport node participating in a Reticulum network will only know the most direct way to get a packet one hop closer to it’s destination."

But could there be a third kind of node or a different type, as in theory the traffic is encrypted, nothing really stops someone from being a transport node and having it load on their system for all the good it will do.  But the idea is a node type that would be able to pass message stores or other data between different parts of the node networks.   

My thought process goes think a bike messenger running packages between buildings.  You have the node running in this mode and you can message / communicate but also transport any messages due to be passed. 

And the reasoning being that building this network in different parts of the world, will not exactly the fastest growth. So a node that can pass messages until the user who holds the proper hash to de-crypt it is found should safely be possible.   But also for in areas like where i live between islands we sometimes have to take a ferry to get home and back.  The region is not exactly friendly to node placement.     So people who have a node on their bag that is powered with a 18650 for instance can without any user effort be messenger nodes and pass messages between network areas.  Though the re-association between different networks may be something to get around.  Either way an idea.

M@

**Comment by markqvist** [comment]
I think you may be confusing Reticulum and LXMF, or lumping them together as one.

Reticulum is a base-layer protocol. It handles reliable and secure data transport from point A to point B. You can think of this as Reticulum being a replacement for Ethernet+IPv4/6+TCP/UDP+TLS.

LXMF is a messaging protocol built on top Reticulum. You can think of this as a replacement for something like email, XMPP or Matrix.

Reticulum requires Transport Nodes (routers that transport traffic for the network) to be physically stationary (in most cases, there are supported configurations of mobile routers, for example in vehicles).

LXMF does not require anything to be stationary. LXMF Propagation Nodes, that host the encrypted message stores, can roam around physically or virtually as much as they want.

Does that clear things up?

> So people who have a node on their bag that is powered with a 18650 for instance can without any user effort be messenger nodes and pass messages between network areas.

Mobile LXMF propagation nodes will already work in a scenario like this.

> Though the re-association between different networks may be something to get around.

Nope, that works transparently and without anything required by the user. One of the brilliant things about Reticulum is that it can transparently and seamlessly handle endpoints moving around to different network segments, or even to different physically isolated networks.

**Reply by ReconPirate** [reply]
Thankyou, that does clear that up, I was mixing the two a bit, But you also answered my question as to transport nodes as well, as to they are more online constantly connected nodes in the network and thus if i have one constantly running on a ups in my car I can run it as a transport node. And a few others on our island to help prop up the network over here.  Perfect ^_^ now that i got that down i'm going to go re-read the manual again and  plot how the hell im going to play with different network setups when the other two liligo lora devices arrive.  But that is so cool how they re-associate transparently,  Thanks for all the hard work on this really cool networking tech.   Also thanks for the quick reply
M@
